Source/Charts/Animation/Animator.swift

```diff
@@ -29,15 +29,65 @@ public protocol AnimatorDelegate
 @objc(ChartAnimator)
 open class Animator: NSObject
```

Comment:
If these changes only apply to the PieChart, then you should subclass `Animator` to be `PieChartAnimator`. If some things still belong in the base class, of course keep them here.